### ISSUE TITLE
fails if "repository" is set to a string

### ISSUE THREAD

**Issue by gr2m** [skypackjs/package-check#5]
See repro at https://gist.github.com/gr2m/b80b53f077407ca0e20dc192c2d6f127

```
git@gist.github.com:b80b53f077407ca0e20dc192c2d6f127.git skypack-package-check-5
cd skypack-package-check-5
npm install
npx package-check
```

But setting `"repository"` to just a string is valid as per https://docs.npmjs.com/files/package.json#repository

**Comment by gr2m** [comment]
here is how we get the repository URL in semantic-release:

https://github.com/semantic-release/semantic-release/blob/3abcbaf2561a208180a1f8eddc1d8a5c1006fe48/lib/get-config.js#L95-L98

**Comment by stramel** [comment]
Can confirm, just brought this up here: https://twitter.com/stramel89/status/1319408395938648065

**Comment by FredKSchott** [comment]
Fixed in 9ca20fa
